extensions/kotlin/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
   api(project(":api:all"))
 
   compileOnly("org.jetbrains.kotlin:kotlin-stdlib-common")
-  compileOnly("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2")
+  compileOnly("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")
```

Comment:
@jkwatson presumably we're pinned to this version for compatibility reasons?

Reply:
I don't remember why we might have done that, TBH. @anuraaga might?

Reply:
Yeah I think so, otherwise this would be in dependencyManagement.